eslint.config.mjs

```diff
@@ -116,7 +116,7 @@ export default [
   {
     name: 'Mocha Linting',
     files: ['test/**', '**/*.cy.js'],
-    ...mocha.configs.flat.recommended,
```

Comment:
In `v11`, they are now only using flat config: https://github.com/lo1tuma/eslint-plugin-mocha/issues/382.